src/System.Net.Http/src/System/Net/Http/HttpClientHandler.FreeBSD.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
```

Comment:
The FreeBSD, Linux, OSX, and Unix versions of this file look identical to me.  You can probably get away with the Unix only one for now, and as the other platforms come online their variant files would have implementations...